converter.go

```diff
@@ -1026,6 +1035,15 @@ func arrowToRecord(record arrow.Record, pool memory.Allocator, rowType []execRes
 						tb.AppendNull()
 					}
 				}
+			} else if col.DataType().ID() == arrow.INT64 {
+				for i, t := range col.(*array.Int64).Int64Values() {
+					if !col.IsNull(i) {
+						val := time.Unix(0, t*int64(math.Pow10(9-int(srcColumnMeta.Scale)))).UTC()
```

Comment:
Other calls to `time.Unix` under `case timestampLtzType` aren't followed by `.UTC()`, is this deviation intentional?

Reply:
Good call!
It's copy and paste mistake. I should follow line 1049.